util/file_reader_writer.h

```diff
@@ -277,10 +300,14 @@ class WritableFileWriter {
   Status SyncInternal(bool use_fsync);
 };
 
-// FilePrefetchBuffer can automatically do the readahead if file_reader,
-// readahead_size, and max_readahead_size are passed in.
+// FilePrefetchBuffer is a smart buffer to store and read data.
+// It can automatically do a readahead if a file_reader, readahead_size, and
+// max_readahead_size are passed in to the constructor.
```

Comment:
Whether features are enabled by passing in constructor or other way, seems to be better as a comment to the constructive.